basic_samples/Authentication/AuthorizationCodeFlow/PowerBI/README.md

```diff
@@ -31,6 +31,26 @@ The OCS Connector for Power BI Desktop is used to get data from the OCS API into
 1. Copy the `.mez` file from either `/bin/Debug` or `/bin/Release` (depending on settings) into the new `Custom Connectors` folder
 1. Start up Power BI Desktop, and the connector should be available
 
+## Building without Visual Studio
+
+Visual Studio builds Custom Connectors into a `.zip` file with a custom file extension, `.mez`. Since these files are in reality simple `.zip` files, it is possible to build or update the connector without using Visual Studio.
+
+To build the connector:
```

Comment:
Might want a version of PowerBI referenced, this seems likely to change one day

Reply:
@derekendres See if this works for you

Reply:
@derekendres This look OK now?